azurerm/data_source_sql_server_test.go

```diff
@@ -0,0 +1,46 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+)
+
+func TestAccDataSourceAzureRMSqlServer(t *testing.T) {
```

Comment:
actually on reflection, could we suffix this `_basic` to match the other tests? (I'll push a commit for this, I hope you don't mind!)